pkg/cedar/intake/models/easi_intake.go

```diff
@@ -22,9 +22,9 @@ type EASIIntake struct {
 	DecisionNextSteps           string `json:"decisionNextSteps" jsonschema:"description=Steps that the business owner should take after receiving their decision,example=Go get a contract"`
 	EaCollaborator              string `json:"eaCollaborator" jsonschema:"description=N/A,example=N/A"` // TODO: doesn't seem like this is ever populated, remove?
 	EaCollaboratorName          string `json:"eaCollaboratorName" jsonschema:"description=Enterprise Architecture (EA) Collaborator,example=John Doe"`
-	EaSupportRequest            string `json:"eaSupportRequest" jsonschema:"enum=,enum=false,enum=true,description=Does the request need EA support,example=True"`
+	EaSupportRequest            string `json:"eaSupportRequest,omitempty" jsonschema:"enum=,enum=false,enum=true,description=Does the request need EA support,example=True"`
 	ExistingContract            string `json:"existingContract" jsonschema:"description=Is there an existing contract for this effort,example=HAVE_CONTRACT"`
-	ExistingFunding             string `json:"existingFunding" jsonschema:"enum=,enum=false,enum=true,description=Will this project be funded out of an existing source,example=True"`
+	ExistingFunding             string `json:"existingFunding,omitempty" jsonschema:"enum=,enum=false,enum=true,description=Will this project be funded out of an existing source,example=True"`
```

Comment:
Type needs to be changed, see note on `easi_biz_case.go`.

Reply:
Updated in https://github.com/CMSgov/easi-app/pull/1553/commits/20aa9583e58c0a8f4495ddb39d59fb80c568250b.